Workiva/wGulp
# Fix minify tasks dependencies

Fixes #91
## Problem

Minify task does not depend on bundle or libraryDist. This causes things in dist to not be minified properly because they aren't necessarily all there when the minify task starts.
## Solution

Add necessary dependencies to taskTree.
## Testing

Simple taskTree sanity checks
- Ensure minify:js task is started after libraryDist and bundle tasks.
- Ensure minify:css task is started after libraryDist task.

@trentgrover-wf 
@evanweible-wf
